content/en/cloud_cost_management/recommendations/_index.md

```diff
@@ -291,6 +249,18 @@ multifiltersearch:
       recommendation_type: Terminate Network Load Balancer
       recommendation_description: A network load balancer with 0 processed bytes.
       recommendation_prerequisites: ""
+    - category: Downsize
+      cloud_provider: AWS
+      resource_type: Lambda
+      recommendation_type: Downsize Lambda Provisioned Concurrency
+      recommendation_description: AWS Lambda function with over-allocated provisioned concurrency.
+      recommendation_prerequisites: ""
+    - category: Terminate
+      cloud_provider: AWS
+      resource_type: CloudWatch Logs
+      recommendation_type: Delete Lambda Cloudwatch Logs and write permissions
+      recommendation_description: Lambda function that can have write CloudWatch Logs permissions removed.
```

Comment:
perhaps:
```suggestion
      recommendation_description: Remove write permissions for Lambda CloudWatch Logs to prevent further unnecessary logging.
```